vnext/Microsoft.ReactNative/ABIViewManager.cpp

```diff
@@ -115,29 +115,31 @@ void ABIViewManager::UpdateProperties(react::uwp::ShadowNodeBase *nodeToUpdate,
 }
 
 folly::dynamic ABIViewManager::GetCommands() const {
-  folly::dynamic innerParent = Super::GetCommands();
```

Comment:
Our superclass always returned nothing, so removing it doesn't do anything in practice, but I don't think this should have been here to begin with.

I.e. we don't dispatch to the superclass, so even if it did has commands to report, we could never actually run them.

Reply:
shouldn't we fix the "dispatching to the superclass isn't working" problem then? Can you file an issue if you can't fix it in this PR?

Reply:
I think it's fine to expect the Base View Manager won't handle anything in this case. Right now the only thing it does is `assert(false)` as a method to catch unhandled commands.